src/Sulu/Bundle/WebsiteBundle/Twig/Navigation/NavigationTwigExtension.php

```diff
@@ -149,6 +150,23 @@ public function breadcrumbFunction($uuid)
     }
 
     /**
+     * @param string $requestUrl
+     * @param string $itemUrl
+     *
+     * @return bool
+     */
+    public function isActiveNavElementFunction($requestUrl, $itemUrl)
+    {
+        if ($requestUrl !== $itemUrl) {
+            if (substr($requestUrl, 0, strlen($itemUrl)) !== $itemUrl) {
```

Comment:
the problem with this is that you also match following two pathes: `/news/2016` `/news/2016-02` i think you have to use a more complicated check or try a regex like this: `/^\/news\/2016\/`maybe thats enogh